library/core/src/cell/once.rs

```diff
@@ -164,6 +164,45 @@ impl<T> OnceCell<T> {
         }
     }
 
+    /// Gets the mutable reference of the contents of the cell,
+    /// initializing it with `f` if the cell was empty.
+    ///
+    /// # Panics
+    ///
+    /// If `f` panics, the panic is propagated to the caller, and the cell
+    /// remains uninitialized.
+    ///
+    /// It is an error to reentrantly initialize the cell from `f`. Doing
```

Comment:
you can delete the reentrant language, since reentrance is prevented by `&mut self` being borrowed and not re-exposed

Reply:
Good catch. Updated.

Reply:
all 4 functions had that comment, you need to remove it from the other three too

Reply:
Updated.